ui/admin/tests/acceptance/auth-methods/list-test.js

```diff
@@ -76,28 +68,28 @@ module('Acceptance | auth-methods | list', function (hooks) {
   test('users can navigate to auth methods with proper authorization', async function (assert) {
     await visit(urls.globalScope);
 
-    await click(`[href="${urls.orgScope}"]`);
+    await click(commonSelectors.HREF(urls.orgScope));
 
     assert.true(
       instances.scopes.org.authorized_collection_actions[
         'auth-methods'
       ].includes('list'),
     );
-    assert.dom(`[href="${urls.authMethods}"]`).exists();
+    assert.dom(commonSelectors.HREF(urls.authMethods)).isVisible();
   });
 
   test('users cannot navigate to index without either list or create actions', async function (assert) {
     instances.scopes.org.authorized_collection_actions['auth-methods'] = [];
     await visit(urls.globalScope);
 
-    await click(`[href="${urls.orgScope}"]`);
+    await click(commonSelectors.HREF(urls.orgScope));
 
     assert.false(
       instances.scopes.org.authorized_collection_actions[
         'auth-methods'
       ].includes('list'),
     );
-    assert.dom(`[href="${urls.authMethods}"]`).doesNotExist();
+    assert.dom(commonSelectors.HREF(urls.authMethods)).doesNotExist();
```

Comment:
should we change this one to isNotVisible too?

Reply:
@DhariniJeeva good catch! I have been changing the `doesNotExists` but definetly missed this one 😅 thanks for the catch!